src/planscape/forsys/rank_projects_for_a_single_scenario.R

```diff
@@ -0,0 +1,44 @@
+suppressMessages({
+  library(dplyr)
+  library(forsys)
+  library(sf)
+})
+
+rank_projects_for_a_single_scenario <- function(input_stand_data,
+                                                priorities,
+                                                priority_weights,
+                                                stand_id_field,
+                                                proj_id_field,
+                                                stand_area_field,
+                                                stand_cost_field) {
+
+  scenario_output_fields <- c(priorities, stand_area_field, stand_cost_field)
+
+  priorities_pcp <- paste(priorities, "_PCP", sep="")
+
+  stand_data <- input_stand_data %>% 
+                forsys::calculate_spm(fields = priorities) %>%
+                forsys::calculate_pcp(fields = priorities)
+  stand_data <- stand_data %>%
+                forsys::combine_priorities(fields = priorities_pcp, 
+                                            weights = priority_weights, 
+                                            new_field = 'preset_priority')
```

Comment:
what does this var name semantically mean? can we add a comment?

Reply:
I've renamed this to forsys_input_data.
It's the input dataframe to forsys.